build-tools/packages/build-cli/src/codeCoverage/index.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export { getCodeCoverageSummary, CodeCoverageSummary } from "./codeCoveragePr.js";
```

Comment:
Don't use barrel files. Import from the modules directly.

Reply:
done